modules/ROOT/pages/storage.adoc

```diff
@@ -354,6 +354,70 @@ storage:
       with_mount_unit: true
 ----
 
+== Adding Network Storage
+
+Fedora CoreOS systems can be configured to mount network file systems such as NFS and CIFS. This is best achieved by using Ignition to create systemd units. File systems can be mounted on boot by creating a standard mount unit. Alternately, a file system can be mounted when users access the mountpoint by creating an additional automount unit. Below are examples of each for an NFS file system.
+
+=== Configuring NFS Mounts
+
+Creating a systemd unit to mount an NFS file system on boot. Note: the .mount file must be named based on the path (e.g. /var/mnt/data = var-mnt-data.mount)
+
+[source,yaml]
+----
+variant: fcos
+version: 1.3.0
+systemd: 
+  units:
+    - name: var-mnt-data.mount
+      enabled: true
+      contents: |
+        [Unit]
+        Description=Mount data directory
+
+        [Mount]
+        What=example.server.edu:/data
+        Where=/var/mnt/data
+        Type=nfs4
+
+        [Install]
+        WantedBy=remote-fs.target
+----
+
+Creating a systemd unit to mount an NFS file system when users access the mount point (automount) 
+[source,yaml]
+----
+variant: fcos
+version: 1.3.0
+systemd: 
+  units:
+    - name: var-mnt-data.mount
+      contents: |
+        [Unit]
+        Description=Mount data directory
+
+        [Mount]
+        What=example.server.edu:/data
+        Where=/var/mnt/data
+        Type=nfs4
+
+        [Install]
+        WantedBy=remote-fs.target
+
+    - name: var-mnt-data.automount
+      enabled: true
+      contents: |
+        [Unit]
+        Description=Automount data directory
+
+        [Automount]
+        TimeoutIdleSec=20min
+        Where=/var/mnt/data
+
+        [Install]
+        WantedBy=multi-user.target
```

Comment:
I _think_ this also wants to be `remote-fs.target`, but I'm not completely sure.  There's a risk of services in `multi-user.target` trying to use the mountpoint before the automounter is up.

Reply:
Is correct.